sources/packages/backend/apps/api/src/services/education-program-offering/education-program-offering-validation.models.ts

```diff
@@ -214,6 +224,10 @@ export enum OfferingValidationWarnings {
   InvalidStudyDatesPeriodLength = "invalidStudyDatesPeriodLength",
   InvalidFundedStudyPeriodLength = "invalidFundedStudyPeriodLength",
   OfferingCostExceedMaximum = "offeringCostExceedMaximum",
+  ProgramNotAviation = "programNotAviation",
+  MatchProgramAviationCredential = "matchProgramAviationCredential",
```

Comment:
Can we rename this to follow the idea from the others? For instance: `programAviationCredentialMismatch`.